hack/lib/util/environment.sh

```diff
@@ -148,7 +149,17 @@ function os::util::environment::setup_tmpdir_vars() {
     HOME="${FAKE_HOME_DIR}"
     export HOME
 
-    mkdir -p  "${BASETMPDIR}" "${LOG_DIR}" "${VOLUME_DIR}" "${ARTIFACT_DIR}" "${HOME}"
+    # ensure that the directories are clean
+    for target in $( ${USE_SUDO:+sudo} findmnt --output TARGET --list ); do
```

Comment:
seeing `openshift/origin/hack/lib/util/environment.sh: line 131: findmnt: command not found` error with this